.github/workflows/pythonbuild.yml

```diff
@@ -109,6 +109,42 @@ jobs:
         with:
           path: ./wheelhouse/*.whl
 
+  build_wheels_aarch64:
```

Comment:
This should be added to the list of required jobs for the job deploy-wheels

Reply:
Added.